nncf/common/pruning/utils.py

```diff
@@ -291,14 +314,15 @@ def count_flops_and_weights_per_node(graph: NNCFGraph,
         else:
             filters_per_channel = num_out_channels // node.layer_attributes.groups
 
-        flops_numpy = 2 * np.prod(node.layer_attributes.kernel_size) * \
-                      num_in_channels * filters_per_channel * np.prod(output_shapes[name])
-        weights_numpy = np.prod(node.layer_attributes.kernel_size) * num_in_channels * filters_per_channel
+        flops_numpy = 2 * np.prod(kernel_size) * num_in_channels * filters_per_channel * np.prod(output_shapes[name])
```

Comment:
Let's discuss flops computing formula. I decided to check it, and as far as I understood here should be `input_shapes` instead of `output_shapes`

Reply:
The formula looks correct. `input_shape` is possible to use, but it will be more complicated with considering `padding` and `stride`.